lib/runner/test-source.js

```diff
@@ -46,11 +47,50 @@ class TestSource {
     return testsource;
   }
 
+  getTestSourceForRerunFailed(argv) {
+    const rerunJsonFile = getRerunFailedFile(argv);
+  
+    try {
+      const {modules = {}} = require(rerunJsonFile);
+      const testsource = [];
+    
+      Object.keys(modules).forEach(moduleKey => {
+        if (modules[moduleKey] && modules[moduleKey].status === 'fail') {
+          testsource.push(modules[moduleKey].modulePath);
```

Comment:
Does this means the reruns will be file specific, not testcase specific? 

@gravityvi thoughts on this?

Reply:
As per our discussions, this is good enough for a first version.